### ISSUE TITLE
Timeline Slider Labels Should Match the Date

### ISSUE THREAD

**Issue by SteveChab** [cmu-delphi/www-covidcast#513]
The timeline slider label numbers seem to be arbitrary. My screen reader (Voiceover) read a series of large numbers ("18396.17, etc.") rather than the date.

![4-18000](https://user-images.githubusercontent.com/275785/94615082-292a4f80-0275-11eb-8dac-162f0e91e6c1.png)

**Comment by sgratzl** [comment]
@SteveChab can you please take a look at https://deploy-preview-514--cmu-delphi-covidcast.netlify.app/ I tried to fix it but somehow I cannot access with the windows narrator screenreader the individual tabs, so I don't know how to test it.

**Comment by SteveChab** [comment]
Terrific!
